config.ts

```diff
@@ -94,6 +94,12 @@ const conf: Config = {
       requiredFields: ["compartment", "availability_domain", "vcn_compartment"],
       relistFields: ["migr_image_map", "migr_image"],
     },
+    {
+      name: "o3c",
```

Comment:
If this is an initials acronym it should be `oc3`.

`o3c` seems to be a [name for a US government cloud region](https://docs.oracle.com/en/cloud/paas/application-integration/oracle-integration-gov/oracle-integration-oci-us-government-cloud.html#GUID-78262B12-EC95-4AD6-A784-88E216D6B7A4).